go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/coreos/go-semver v0.2.0
 	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
 	github.com/denisenkom/go-mssqldb v0.0.0-20200428022330-06a60b6afbbc
+	github.com/dgrijalva/jwt-go/v4 v4.0.0-preview1 // indirect
```

Comment:
If you run a `go mod tidy`, do the indirect deps go away?

Reply:
Indeed, sorry!